modules/tinymce/src/plugins/quickbars/test/ts/browser/ContentEditableTest.ts

```diff
@@ -57,4 +57,11 @@ describe('browser.tinymce.plugins.quickbars.ContentEditableTest', () => {
     TinySelections.select(editor, 'p span', []);
     await pAssertToolbarNotVisible();
   });
+
+  it('TINY-9190: Toolbar is not shown in the fake caret', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p contenteditable="false">CEF element</p>');
+    // Selection is already in the fake caret
```

Comment:
Maybe this should also assert that the fake caret is actually showing or that the `editor.selection.getRng` returns the position before the `p` tag though TinyAssertions. Since now the test kind of just assumes that the caret will be where the comment says it is.